src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -257,7 +257,7 @@ PARAM_DEFINE_FLOAT(MPC_Z_V_AUTO_DN, 1.f);
  * @decimal 1
  * @group Multicopter Position Control
  */
-PARAM_DEFINE_FLOAT(MPC_Z_VEL_MAX_DN, 1.f);
+PARAM_DEFINE_FLOAT(MPC_Z_VEL_MAX_DN, 1.5f);
```

Comment:
I don't see a problem with this since we're not increasing the value too high where wake turbulence can get a big issue. Also, the landing should never be done by putting normal mission waypoints into the ground or fully pulling the stick down but with a land waypoint or manually commanding a gentle touch down. And even if it's misused I don't see how a 1.5 m/s touch down destroys a typical multi-copter while it does not damage a typical VTOL.